crates/storage/src/vm_storage.rs

```diff
@@ -367,12 +369,15 @@ where
 
         // verify key is in range
         current_key
-            .checked_add(U256::from(values.len()))
+            .checked_add(U256::from(values.len().saturating_sub(1)))
```

Comment:
can we use `checked_sub` instead?

Reply:
I don't want to because if values is 0 then I don't want to throw and error or anything I just want the checked_add to not do anything